src/playlist-selectors.js

```diff
@@ -352,3 +355,37 @@ export const minRebufferMaxBandwidthSelector = function(settings) {
 
   return rebufferingEstimates[0] || null;
 };
+
+/**
+ * Chooses the appropriate media playlist, which in this case is the lowest bitrate
+ * one with video.  If no renditions with video exist, return the lowest audio rendition.
+ *
+ * Expects to be called within the context of an instance of HlsHandler
+ *
+ * @return {Object|null}
+ *         {Object} return.playlist
+ *         The lowest bitrate playlist that contains a video codec.  If no such rendition
+ *         exists pick the lowest audio rendition.
+ */
+export const lowestBitrateCompatibleVariantSelector = function() {
+  const playlists = this.playlists.master.playlists.slice();
```

Comment:
could replace this `slice` call with the enabled filter, then you wouldn't need to do it later or have the local const